### ISSUE TITLE
No account info in CLI

### ISSUE THREAD

**Issue by maaktweluit** [golemfactory/clay#2217]
Running b0.12.0 on ubuntu server:
```
$ golemcli account info
[16:22:20.155001] ERROR: Exception: too many values to unpack (expected 3)`
```

There are no related errors in the logs

**Comment by maaktweluit** [comment]
@mfranciszkiewicz applied a patch to solve this in a recent PR to the release branch
https://github.com/golemfactory/golem/pull/2236/files#diff-b9f07081bef4581763153e7f1f60fdf8R28

Please double check if its working on all OS'es after the release is merged back

**Comment by Wiezzel** [comment]
Closed by #2236 
Tested on Ubuntu, Windows, and macOS.
